src/component/home/index.tsx

```diff
@@ -32,9 +33,37 @@ export function Component(props: HomeProps) {
     timelineId,
     userId,
   };
+
+  const audienceId = '1';
+  const timelineSearch: ITimelineQuery = {
+    audienceId,
+    userId,
+    organizationId,
+  };
+
+  const { data: timelinesData, isSuccess } = useTimelines(timelineSearch);
+
+  
+  const allUpdatesSearch: any = {
+    organizationId,
+    timelineId,
+    userId,
+    timelines:timelinesData,
+  };    
+
+  const { data: allUpdates } = useAllUpdates(allUpdatesSearch);
+  
+
+
```

Comment:
This here indicates that your editor is not setup correctly. When you get your prettier integration sorted out you can maybe 10x again. Also the build is broken due to your current setup.

Reply:
Yup - I don't know what's incorrectly setup. I have it on my todo list to get more familiar with vs code but won't get to it until after Monday

Reply:
When you have some file open and you have the prettier extension installed you should be able to select the formatter in your status bar at the bottom of VSCode. Often the editor does not know what to do if you have multiple formatter options or none selected for a specific file type.